components/eam/src/physics/crm/samxx/pre_timeloop.cpp

```diff
@@ -156,6 +156,9 @@ void pre_timeloop() {
   YAKL_SCOPE( t_vt                    , :: t_vt );
   YAKL_SCOPE( q_vt                    , :: q_vt );
   YAKL_SCOPE( use_VT                  , :: use_VT );
+  YAKL_SCOPE( crm_input_u_vt          , :: crm_input_u_vt);
+  YAKL_SCOPE( u_vt_tend               , :: u_vt_tend);
+  YAKL_SCOPE( u_vt                    , :: u_vt);
```

Comment:
these three lines can also be reorganized so that u can be together with t & q

Reply:
fixed